### ISSUE TITLE
Error trying to start portainer "cannot find cgroup mount destination"

### ISSUE THREAD

**Issue by sheel** [WDCommunity/wdpksrc#97]
Hello, I was able to download the .zip above and install on my PR4100, firmware 5.15.106.

However, when I go to configure, it's an empty page (:9000).

I ssh'd into the PR4100, and there is nothing listening on :9000 and no docker processes running.

**netstat**
> root@illmatic docker # netstat -tulpn | grep 9000
> root@illmatic docker # 

**docker ps**
> root@illmatic docker # docker ps
> CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
> root@illmatic docker # 

**docker is running**
> root@illmatic docker # ls -l /var/run/docker.sock
> srw-rw----    1 root     root             0 Aug  9 23:49 /var/run/docker.sock

**I tried running the portainer command, and get a cgroup mount error**

> root@illmatic docker # docker run -d -p 9000:9000 --restart always -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/portainer:/data portainer/portainer
> 73b72f8dfd01384bde20af579e86d904367bb807ccee69ff777768f333eec0f3
> docker: Error response from daemon: cgroups: cannot find cgroup mount destination: unknown.

**docker info**
> root@illmatic docker # docker info
> Client:
>  Debug Mode: false
> 
> Server:
>  Containers: 4
>   Running: 0
>   Paused: 0
>   Stopped: 4
>  Images: 1
>  Server Version: 19.03.8
>  Storage Driver: vfs
>  Logging Driver: json-file
>  Cgroup Driver: cgroupfs
>  Plugins:
>   Volume: local
>   Network: bridge host ipvlan macvlan null overlay
>   Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
>  Swarm: inactive
>  Runtimes: runc
>  Default Runtime: runc
>  Init Binary: docker-init
>  containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
>  runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
>  init version: fec3683
>  Security Options:
>   seccomp
>    Profile: default
>  Kernel Version: 4.14.22
>  OSType: linux
>  Architecture: x86_64
>  CPUs: 4
>  Total Memory: 3.708GiB
>  Name: illmatic
>  ID: ZXDP:4LW5:ROQL:V4ZT:LZIJ:SYRC:JAVL:DBXP:SN4Q:QMS3:67GS:PUZS
>  Docker Root Dir: /mnt/HD/HD_a2/Nas_Prog/_docker
>  Debug Mode: false
>  Registry: https://index.docker.io/v1/
>  Labels:
>  Experimental: false
>  Insecure Registries:
>   127.0.0.0/8
>  Live Restore Enabled: false
>  Product License: Community Engine
> 
> WARNING: No memory limit support
> WARNING: No swap limit support
> WARNING: No kernel memory limit support
> WARNING: No kernel memory TCP limit support
> WARNING: No oom kill disable support
> WARNING: No cpu cfs quota support
> WARNING: No cpu cfs period support
> WARNING: No cpu shares support

**docker version**
> root@illmatic cgroup # docker version
> Client: Docker Engine - Community
>  Version:           19.03.8
>  API version:       1.40
>  Go version:        go1.12.17
>  Git commit:        afacb8b7f0
>  Built:             Wed Mar 11 01:22:56 2020
>  OS/Arch:           linux/amd64
>  Experimental:      false
> 
> Server: Docker Engine - Community
>  Engine:
>   Version:          19.03.8
>   API version:      1.40 (minimum version 1.12)
>   Go version:       go1.12.17
>   Git commit:       afacb8b7f0
>   Built:            Wed Mar 11 01:30:32 2020
>   OS/Arch:          linux/amd64
>   Experimental:     false
>  containerd:
>   Version:          v1.2.13
>   GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
>  runc:
>   Version:          1.0.0-rc10
>   GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
>  docker-init:
>   Version:          0.18.0
>   GitCommit:        fec3683

_Originally posted by @sheel in https://github.com/WDCommunity/wdpksrc/issues/57#issuecomment-895793310_

**Comment by sheel** [comment]
ok, interesting.

Seems the initial install worked properly and I was able to get `portainer` started. I rebooted the device, but then had to manually start portainer via ssh -

> docker run -d -p 9000:9000 --restart always --name portainer -v /var/run/docker.sock:/var/run/docker.sock -v $(readlink -f ${APKG_PATH})/portainer:/data portainer/portainer

That being said, while trying to debug, I ran some of the scripts by logging into ssh (e.g. daemon setup, install, etc), the scripts seemed not to pick up / build the variables correctly and created directories in root - for example,`/docker`.
